vercel.json

```diff
@@ -4,6 +4,11 @@
 			"destination": "https://opencollective.com/boston-ts-club",
 			"permanent": true,
 			"source": "/donate"
+		},
+		{
+			"destination": "https://docs.google.com/forms/d/e/1FAIpQLSevxqyXjEE2AQ8C7xxkmoiiasr5ONIvPyAl8ad-MseNZ2ZBxA/viewform",
+			"permanent": false,
```

Comment:
That's fine, I debated as well and leaned the other direction, but I'm happy either way.

In that case, care to make `/donate` non-permanent as well?

Reply:
Do you see the donate link changing? I just see the feedback link changing more often than not

Reply:
Heh, I don't see either changing but I'm not known for my accurate predictions lol.

I'm happy either way tbh. @JoshuaKGoldberg does the donate = permanent / feedback = non-permanent work for you?

Reply:
Let's just go with this so it's available tonight. Nothing we can't fix later.